x-pack/plugins/security_solution/public/dashboards/pages/landing_page/index.tsx

```diff
@@ -20,7 +20,25 @@ import * as i18n from './translations';
 import { METRIC_TYPE, TELEMETRY_EVENT, track } from '../../../common/lib/telemetry';
 import { DASHBOARDS_PAGE_TITLE } from '../translations';
 import { useCreateSecurityDashboardLink } from '../../hooks/use_create_security_dashboard_link';
-import { DashboardsTable } from '../../components/dashboards_table';
+import { useGetSecuritySolutionUrl } from '../../../common/components/link_to';
+import type { TagReference } from '../../context/dashboard_context';
+import { useSecurityTags } from '../../context/dashboard_context';
+
+const getInitialFilterString = (securityTags: TagReference[] | null | undefined) => {
+  if (!securityTags) {
+    return;
+  }
+  const uniqueQueryArray = securityTags?.reduce<string[]>((acc, { name }) => {
+    const nameString = `"${name}"`;
+    if (name && acc.indexOf(nameString) === -1) {
+      acc.push(nameString);
+    }
+    return acc;
+  }, []);
+
+  const query = [uniqueQueryArray].join(' or');
```

Comment:
I don't understand this logic. You are calling `join` on an array with only one element. Are the `[ ]` intended?